cilium-sysdump/sysdumpcollector.py

```diff
@@ -208,7 +208,9 @@ def collect_gops_per_pod(self, podstatus, type_of_stat):
                 container,
                 utils.get_current_time(),
                 type_of_stat)
-            cmd = "kubectl exec -n {} {} -c {} -- /bin/gops {} 1 > {}/{}" \
+            cmd = "kubectl exec -n {} {} -c {} -- /bin/sh -c " \
+                "'/bin/gops {} $(cat /var/run/cilium/cilium.pid)'" \
```

Comment:
This will only work for cilium. The cilium-sysdump tool also collects gops stats for hubble standalone (and will for hubble-relay). We need a different approach to fix this issue.